client/X11/xf_rail.c

```diff
@@ -470,7 +470,8 @@ static BOOL xf_rail_window_common(rdpContext* context, WINDOW_ORDER_INFO* orderI
 
 	if (fieldFlags & WINDOW_ORDER_FIELD_TITLE)
 	{
-		xf_SetWindowText(xfc, appWindow, appWindow->title);
+		if (appWindow->title)
```

Comment:
@hardening not sure if it make sense but maybe we should handle this case directly in xf_SetWindowText